src/lldpd/patch/0010-Ported-fix-for-length-exceeded-from-lldp-community.patch

```diff
@@ -0,0 +1,35 @@
+From fdb789c348fdcde6d5ef8b837d7f33718bc0862b Mon Sep 17 00:00:00 2001
+From: sudhanshukumar22 <sudhanshu.kumar@broadcom.com>
+Date: Mon, 23 Nov 2020 20:47:28 -0800
+Subject: [PATCH] Ported fix for https://github.com/lldpd/lldpd/issues/408 from
+ community
```

Comment:
this pr needs to be formated properly, why not use original commit id from the lldp repo?

https://github.com/lldpd/lldpd/commit/78243478dcec865486b387cf361783dc137015b5

Reply:
@lguohan , please note that tests/integration/test_lldpcli.py which is part of the changes in lldpd/lldpd@7824347, is not present in lldp source code repository. So, we cannot take that patch as is. 
I tried taking the patch as is. The error logs are as follows:
error: tests/integration/test_lldpcli.py: does not exist in index
stg import: Diff does not apply cleanly
make[1]: *** [Makefile:19: /sonic/target/debs/buster/lldpd_1.0.4-1_amd64.deb] Error 2
make[1]: Leaving directory '/sonic/src/lldpd'
[  FAIL LOG END  ] [ target/debs/buster/lldpd_1.0.4-1_amd64.deb ]
make: *** [slave.mk:468: target/debs/buster/lldpd_1.0.4-1_amd64.deb] Error 1
Makefile.work:284: recipe for target 'target/debs/buster/lldpd_1.0.4-1_amd64.deb' failed
make[1]: *** [target/debs/buster/lldpd_1.0.4-1_amd64.deb] Error 2
make[1]: Leaving directory '/projects/csg_sonic/sk408845/sonic1/sonic/sonic-buildimage'
Makefile:7: recipe for target 'target/debs/buster/lldpd_1.0.4-1_amd64.deb' failed
make: *** [target/debs/buster/lldpd_1.0.4-1_amd64.deb] Error 2

Reply:
understood that, what I meant is that you can get the patch description format properly, use the original one.

```
lib: remove limit on system description length

The limit was introduced in 9c49ced while fixing a memory leak.
The state data is used to ensure we don't interleave operations. We
need to handle the case where the value is truncated because it is
larger than the allocated size.

Fix #408.
```

Reply:
@lguohan :done, please approve.